hw3_bem_graphics_design/responsive_layout/app/scss/style.scss

```diff
@@ -0,0 +1,468 @@
+  }
+
+  &__inner-diagram {
+    margin-top: 22px;
+  }
+
+  &__inner-img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  &__social-media {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  &__social-media-container {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: center;
+    padding: 12px;
+  }
+
+  &__social-media-name {
+    padding-left: 8px;
+  }
+
+  &__social-media-circle {
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
+  }
+
+  &__social-media-circle--instagram {
+    background: #5a9cfe;
+  }
+
+  &__social-media-circle--facebook {
+    background: #20c997;
+  }
+
+  &__social-media-name {
+    @extend %default-font;
+    font-size: 14px;
+    line-height: 22px;
+    color: $grey;
+  }
+}
+
+.datatable {
+  margin: auto 1.6vw;
+  max-width: 1400px;
+
+  &__top-bar {
+    height: 62px;
+    background: $white;
+    border-bottom: 1px solid $default-border-color;
+    justify-content: space-between;
+    align-items: center;
+    display: flex;
+    padding: 0px 24px;
+    border-top-left-radius: 12px;
+    border-top-right-radius: 12px;
+  }
+
+  &__top-bar-title {
+    @extend %title-font;
+    font-size: 18px;
+    line-height: 30px;
+    color: $light-black;
+  }
+
+  &__top-bar-btn {
+    border: none;
+    background-color: transparent;
+    position: relative;
+    cursor: pointer;
+  }
+
+  &__inner {
+    @extend %default-font;
+    font-size: 14px;
+    line-height: 22px;
+    color: $grey;
+  }
+
+  &__items {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    background: #f9fafb;
+    align-items: center;
+    height: 54px;
+    padding: 0px 24px;
+  }
+
+  &__body {
+    background: $white;
+    padding: 0px 24px;
+  }
+
+  &__row {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    align-items: center;
+    border-bottom: 1px solid $default-border-color;
+  }
+
+  &__source-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  &__name {
+    @extend %title-font;
+    font-size: 14px;
+    line-height: 22px;
+    color: $light-black;
+  }
+
+  &__condition-container {
+    border-radius: 100px;
+    display: flex;
+    align-content: center;
+    padding: 2px 10px;
+    width: 62px;
+    height: 24px;
+    justify-content: center;
+  }
+
+  &__condition-container--first,
+  &__condition-container--forth,
+  &__condition-container--fifth {
+    color: #20c997;
+    background: #dcfff5;
+  }
+
+  &__condition-container--second {
+    color: #fd7e14;
+    background: #ffead8;
+  }
+
+  &__condition-container--third {
+    color: #5f6980;
+    background: #f2f4f7;
+  }
+
+  &__pagination {
+    height: 98px;
+    background: $white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__pagination-list {
+    text-decoration: none;
+    color: $grey;
+    padding: 10px;
+  }
+}
+
+.sidebar {
+  height: 1024px;
```

Comment:
why? `100vh`

Reply:
User expects to see the full sidebar on the screen and page scroll should not affect the sidebar